### DISCUSSION TITLE
Uno Figma Questions

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#13407] [Q&A]
_[From Discord:](https://discord.com/channels/372137812037730304/550416151172087808/1144542602465652896)_

I've been handed a XAML file from Figma for Uno Platform, but when added to the page the layout is near but not great.  It's using utu:AutoLayout to position.

I've seen several reference when looking for info about the component being UWP, but it is a WinUI Uno Platform project that I created in VS 2022.  Is the Xaml from Uno/Figma plugin WinUI project compatible or is it UWP only?

I found a YouTube video of someone walking through the process and whilst only 5 months old it looks like the templates have moved on as the Uno Platform App template he uses is UWP, the same (named) template now is WinUI.  Choosing a UWP template doesn't provide the Uno config dialog after you've specified the solution name.

If all of this sounds like I don't know what I'm doing, that's because I don't - hadn't heard of Uno before this week!

**Comment by francoistanguay** [comment]
Creating a new Uno app does indeed now default to WinUI.
Any XAML coming from the Uno/Figma Plugin should work both for UWP and WinUI.

If there's an example of some markup that can be shared of something that doesnt work or doesnt mimic pixel-perfectly the design, feel free to open an issue or share privately.
